python/paddle/fluid/optimizer.py

```diff
@@ -840,33 +840,45 @@ class ModelAverage(Optimizer):
     """
 
     def __init__(self,
-                 params_grads,
-                 average_window_rate,
+                 average_window_rate=0.15,
```

Comment:
Do not set `0.15` as a default value.  0.15 is not general rate.

Reply:
Done.